src/prefect/server/schemas/actions.py

```diff
@@ -585,6 +585,11 @@ class DeploymentFlowRunCreate(ActionBaseModel):
             " has already been created, the existing flow run will be returned."
         ),
     )
+    labels: Union[schemas.core.KeyValueLabels, None] = Field(
```

Comment:
i think we should use `KeyValueLabels` from `prefect.types` here instead of indirection